packages/cache/src/internal/cacheUtils.ts

```diff
@@ -123,3 +123,11 @@ export function assertDefined<T>(name: string, value?: T): T {
 
   return value
 }
+
+export function isFeatureAvailable(): boolean {
+  if (process.env['ACTIONS_CACHE_URL']) {
+    return true
+  }
+
+  return false
+}
```

Comment:
```suggestion
export function isFeatureAvailable(): boolean {
  return !!process.env['ACTIONS_CACHE_URL']
}
```

☝️ this is a common JavaScript idiom for returning a value's truthiness as a bool